compiler/record-minmax/driver/Driver.cpp

```diff
@@ -127,8 +136,24 @@ int entry(const int argc, char **argv)
   {
     auto input_data_path = arser.get<std::string>("--input_data");
 
-    // Profile min/max while executing the given input data
-    rmm.profileData(mode, input_data_path, min_percentile, max_percentile);
+    if (input_data_format == "h5" || input_data_format == "hdf5")
+    {
+      // Profile min/max while executing the H5 data
+      rmm.profileData(mode, input_data_path, min_percentile, max_percentile);
+    }
+    // input_data is a text file where the absolute path of each record (a representative data) is
+    // written line-by-line. Each record should contain all inputs of a model, concatenated in the
```

Comment:
`record` here gives better understanding but still, it's a bit hard to imagine

```
input data is a text file having file path in each line.
each data file is composed of multiple sets of input data of the model,
concatenated  in same order with the input index of the model

[input 1 data][input 2 data]...[input n data] | [input 1 data][input 2 data]...[input n data] ...
|<-----------   set 1    ------------------>|   |<-----------   set 2    ------------------>|
```

Reply:
According to the picture, a data file should contain a single set (composed of all inputs of a model).

I've added a picture for easier understading. PTAL.